package.json

```diff
@@ -3,16 +3,32 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "coveralls": "^3.0.2",
+    "axios": "^0.18.0",
     "classnames": "^2.2.6",
+    "coveralls": "^3.0.2",
     "dotenv-webpack": "^1.5.7",
+    "enzyme": "^3.6.0",
```

Comment:
shouldn't this be in developer dependencies?

Reply:
I had it in the **`devDependencies`**, but the linter complained that it should be in the **`dependencies`** section. That's why I have it there. **Should I put it back to the devDependencies?**

Reply:
I have moved these to devDependencies.